### DISCUSSION TITLE
Not able to complete my discount application

### DISCUSSION THREAD

**Discussion by SahilDabhi** [community/community#116683] [GitHub Education]
### Hi everyone,

I have uploaded the proof for my college however when i try to add my college name it does not allow me to continue the process after selecting my college name
i am not able to upload the video here let me know which site i can use to upload the video for the following issue

**Comment by samus-aran** [comment]
Hi @SahilDabhi 👋 

We can see that your current application is Approved. 👍  Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

**Reply by SahilDabhi** [reply]
Hey there,
i have been verified however i am not able to select my  school and proceed forward
I am not able to continue from here
i dont know if its okay to post personal details or not so i have scrolled the page down to hide personal info.

![image](https://github.com/community/community/assets/80204672/41401294-8e88-488c-a6d3-51843d404592)
